sickbeard/clients/generic.py

```diff
@@ -1,16 +1,20 @@
 # coding=utf-8
 
+from __future__ import unicode_literals
+
 import re
 import time
 from hashlib import sha1
 from base64 import b16encode, b32decode
+import traceback
 
-import sickbeard
-from sickbeard import logger, helpers, db
-from bencode import bencode, bdecode
+from six.moves.http_cookiejar import CookieJar
```

Comment:
`six` is a python compatibility library.  It's because some standard libs have changed name between python 2 and python 3.  This is part of making Medusa python 3 compliant.